vignettes/rls.Rmd

```diff
@@ -144,20 +131,25 @@ rls_check_status(con, "passwd")
 
 
 ``` r
-(policy1 <- rls_construct_policy(
-  name = "admin_all",
-  table = "passwd",
-  role = "admin",
-  using = "(true)",
-  check = "(true)"
-))
-#> <rls_policy>
-#>   policy name: admin_all
-#>   table: passwd
-#>   role: admin
-#>   using: (true)
-#>   check: (true)
-rls_create_policy(con, policy1)
+policy1 <- rls_tbl(con, "passwd") %>%
+  row_policy("admin_all") %>%
+  rows_existing(TRUE) %>%
+  rows_new(TRUE) %>%
+  to(admin)
+policy1
+#> <row_policy> admin_all
+#>   user: admin
+#>   existing rows: TRUE
+#>   new rows: TRUE
+#> # Source:   SQL [3 x 8]
+#> # Database: postgres  [schambe3@/tmp:5432/schambe3]
+#>     gid home_dir    home_phone   pwhash real_name shell       uid user_name
+#>   <int> <chr>       <chr>        <chr>  <chr>     <chr>     <int> <chr>    
+#> 1     0 /root       111-222-3333 xxx    Admin     /bin/dash     0 admin    
+#> 2     1 /home/bob   123-456-7890 xxx    Bob       /bin/zsh      1 bob      
+#> 3     1 /home/alice 098-765-4321 xxx    Alice     /bin/zsh      2 alice
+rls_run(policy1)
+#> [1] 0
```

Comment:
Why does this return `0`? Asking because `0` usually means failure, but this should succeed.

Reply:
it's a return value from `dbExecute` - see https://dbi.r-dbi.org/reference/dbExecute.html#value - so the number of rows affected. Thoughts on a good return object for this fun? Maybe a tibble? or just invisible?